src/Features/CSharp/Portable/Completion/KeywordRecommenders/CheckedKeywordRecommender.cs

```diff
@@ -18,10 +20,29 @@ public CheckedKeywordRecommender()
 
         protected override bool IsValidContext(int position, CSharpSyntaxContext context, CancellationToken cancellationToken)
         {
-            return
-                context.IsStatementContext ||
+            if (context.IsStatementContext ||
                 context.IsGlobalStatementContext ||
-                context.IsNonAttributeExpressionContext;
+                context.IsNonAttributeExpressionContext)
+            {
+                return true;
+            }
+
+            var targetToken = context.TargetToken;
+
+            if (targetToken.Kind() == SyntaxKind.OperatorKeyword)
+            {
+                if (targetToken.GetPreviousToken(includeSkipped: true).IsLastTokenOfNode<TypeSyntax>())
+                {
+                    return true;
```

Comment:
nit: consider adding comments to illustrate the situation (here and below)

Reply:
note: i would prefer this as well.  but it's not a problem to not have it.